### PR TITLE
Add data_type function and unit test

### PR DESCRIPTION
**Pull request by jasmineqyj**
Hey @gptzjs, data_type.py and test_data_type.py are ready for you to review!
Thanks.

Issues: 
#28 create data_type function
#29 create data_type function unit test

### CODE REVIEW

File: PrepPy/data_type.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        Original feature dataframe containing one column for each feature.
+
+    Returns
+    -------
+    dict
+        Stores the categorical and numerical columns separately as two dataframes.
+
+    Examples
+    --------
+    >>> from prepPy import prepPy as pp
```
Comment (gptzjs) [inline PrepPy/data_type.py]:
```suggestion
    >>> from PrepPy import PrepPy as pp
```

**Comment by gptzjs** [comment]
@jasmineqyj Forgot to mention that the test sand the try except are both OK!

**Comment by jasmineqyj** [comment]
Hi @gptzjs, Thank you for your timely feedbacks! You are right about the `my_data` not being separated properly and I think that's due to the way I defined it.

In my docstring, I wrote:
`pd.DataFrame(np.array([['apple', 3, 0], ['banana', 5, 6], ['pear', 8, 9]]),
                               columns=['fruits', 'count', 'price'])`
to define a DataFrame, and this will make everything in `['apple', 3, 0]` as type `object` instead of `object, integer, integer`.

In real life, I think it's rare to see data defined this way (row by row). I also tried some other data like `spotify` or `breast_cancer` and they worked well.

So I think I should update my docstrings instead? Let me know if you have any other questions!

**Comment by gptzjs** [comment]
@jasmineqyj OK that makes sense, thanks for the explanation!

**Comment by gptzjs** [comment]
Sorry forgot to merge earlier

**Pull request merged**